WordPress/UITestsFoundation/Globals.swift

```diff
@@ -24,6 +24,17 @@ public func navigateBack() {
     }
 }
 
+// Generic function for cases where ScreenObject times out due to long loading time
+// making the Editor Screen evaluate to 'nil'
```

Comment:
Nitpick: If you want to make code stand out in comments, backticks are more appropriate.

```suggestion
// making the Editor Screen evaluate to `nil`
```

I say this because backticks parse as inline code in Markdown.

In fact, if we were to make that comment a documentation comment, Xcode would render them like this:

<img alt="image" src="https://user-images.githubusercontent.com/1218433/152979699-c5a54f2f-6455-4d00-9dd5-bae9a9747c80.png" width=400/>

vs with the single quotes:

<img alt="image" src="https://user-images.githubusercontent.com/1218433/152979736-e3d51e25-540d-4cac-a98a-fdf9181a966f.png" width=400/>